system/include/compat/emmintrin.h

```diff
@@ -383,8 +383,9 @@ _mm_cvtpd_epi32(__m128d __a)
   int m[2];
   for(int i = 0; i < 2; ++i)
   {
-    int x = lrint(__a[i]);
-    if (x != 0 || fabs(__a[i]) < 2.0)
+    double e = __a[i];
+    int x = lrint(e);
+    if ((x != 0 || fabs(e) < 2.0) && !isinf(e) && e <= LLONG_MAX && e >= LLONG_MIN)
```

Comment:
should this be INT_MAX? we are still converting to int32 here, not int64?

Reply:
good catch!